src/current/v24.3/data-resilience.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Data Resilience
+summary: An overview of how CockroachDB deployments can achieve resilience with built-in features and tooling.
+---
+
+CockroachDB provides built-in [_high availability_](#high-availability) features and [_disaster recovery_](#disaster-recovery) tooling to achieve operational resiliency in various deployment topologies and use cases. To maximize uptime, high availability features ensure continuous access to data without interruption even in the presence of failures or disruptions. Disaster recovery tools allow for recovery from major incidents to minimize downtime and data loss.
+
+<image src="{{ 'images/v24.3/resilience.svg' | relative_url }}" alt="Diagram showing how high availability features and disaster recovery tools create a resilient CockroachDB deployment." style="max-width:100%" />
+
+You can balance required SLAs and [recovery objectives]({% link {{ page.version.version }}/disaster-recovery-overview.md %}) with the cost and management of each of these features to build a resilient deployment.
+
+{{site.data.alerts.callout_success}}
+For a practical guide on how CockroachDB uses Raft to replicate, distribute, and rebalance data, refer to the [Replication and Rebalancing]({% link {{ page.version.version }}/demo-replication-and-rebalancing.md %}) demo.
+{{site.data.alerts.end}}
+
+## High availability
+
+- [**Raft replication**]({% link {{ page.version.version }}/architecture/replication-layer.md %}): CockroachDB's built-in Raft replication stores data safely and consistently on multiple nodes to ensure no downtime even during a temporary node outage. [Replication controls]({% link {{ page.version.version }}/configure-replication-zones.md %}) allow you to configure the number and location of replicas to suit a deployment.
+- [**Fault tolerance**]({% link {{ page.version.version }}/demo-fault-tolerance-and-recovery.md %}): Capabilities built in to CockroachDB to perform routine maintenance operations without any affect on performance. For example, [online schema changes]({% link {{ page.version.version }}/online-schema-changes.md %}), [WAL failover]({% link {{ page.version.version }}/cockroach-start.md %}#write-ahead-log-wal-failover).
```

Comment:
suggest changing "without any affect on" to something more like "with minimal impact to foreground performance" or so

Reply:
If it doesn't violate our style guide, I think it's helpful to spell out write ahead log (WAL) failover.

Reply:
Done, both, good points!